scala-interpreter/src/main/scala/org/apache/toree/kernel/interpreter/scala/ScalaInterpreter.scala

```diff
@@ -99,6 +99,33 @@ class ScalaInterpreter(private val config:Config = ConfigFactory.load) extends I
      this
    }
 
+  /**
+   * Start initialization with only the given config
+   *
+   * @return this partially initialized scalaInterpreter
+   */
+  def startInit(): ScalaInterpreter = {
+    import scala.collection.JavaConverters._
+    val args = config.getStringList("interpreter_args").asScala.toList
+    settings = newSettings(args)
+    settings = appendClassPath(settings)
+
+    start()
```

Comment:
`start` is being called in `init` too. Is this potentially problematic? If this is the same `start` from `ScalaInterpreterSpecific`, won't this throw an exception due to the require? Actually, I think I see what's going on. It appears you don't call the init method anymore.

Reply:
yep.